project_resource_calendar/models/calendar_event.py

```diff
@@ -27,11 +27,21 @@ class CalendarEvent(models.Model):
         string='Floor',
         related='room_id.floor',
     )
+    allow_double_book = fields.Boolean(
```

Comment:
Not sure if this is the best name for the field. is it? Because we are checking for the double booking of the room. But if it is tightly related with the double book of the event it might be ok... so this is just a comment.